google/resource_sql_database_instance.go

```diff
@@ -56,6 +56,13 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 							Optional: true,
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
+						"availability_type": &schema.Schema{
```

Comment:
This can only have two possible values (REGIONAL and ZONAL), right? Let's add a ValidationFunc for that (`validation.StringInSlice([]string{"REGIONAL", "ZONAL"}, false)`)